config/config.exs

```diff
@@ -44,11 +44,19 @@ config :fz_http, FzHttpWeb.Endpoint,
 
 config :fz_http,
   wireguard_ipv4_enabled: true,
-  wireguard_ipv4_network: "100.64.0.0/10",
-  wireguard_ipv4_address: "100.64.0.1",
+  wireguard_ipv4_network: %{__struct__: Postgrex.INET, address: {100, 64, 0, 0}, netmask: 10},
```

Comment:
This is on purpose: struct is not available when Elixir reads the config, so we create it using `__struct__` and module name.

Reply:
I figured. Nice workaround!